apis/kueue/webhooks/clusterqueue_webhook.go

```diff
@@ -38,13 +38,16 @@ const (
 	isNegativeErrorMsg string = `must be greater than or equal to 0`
 )
 
-type ClusterQueueWebhook struct{}
+type ClusterQueueWebhook struct {
+	scheme *runtime.Scheme
```

Comment:
where is this used?

Reply:
Left over from another approach I tried. Removed